### PR TITLE
Add support for Swift 5.4

### PR DESCRIPTION
**Pull request by Luke-Roy-IBM**
Added suport for Swift 5.4 by copying and adapting core of Swift 5.3. 
Files Modified in the core during this Process (version set to 5.4 from 5.3):
- core/swift54Action/build.gradle
- core/swift54Action/CHANGELOG.md
- core/swift54Action/Dockerfile

Added Swift 5.4 entries to:
- settings.gradle
- .travis.yml
- README.md
- ansible/files/runtimes.json

### CODE REVIEW

File: core/swift54Action/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+# Apache OpenWhisk Swift 5.4 Runtime Container
+
+## 1.16.0
```
Comment (dgrove-oss) [inline core/swift54Action/CHANGELOG.md:22]:
```suggestion
## Next Release
```

Comment (dgrove-oss) [reply core/swift54Action/CHANGELOG.md:22]: can't be 1.16.0 because we already released it.

**Review by dgrove-oss — APPROVED** [review]
One nitpick comment; otherwise lgtm.  Thanks @Luke-Roy-IBM !

File: core/swift54Action/CHANGELOG.md (hunk 1 above)
Comment (Luke-Roy-IBM) [reply core/swift54Action/CHANGELOG.md:22]: Thanks for the hint :D @dgrove-oss

**Comment by dgrove-oss** [comment]
@Luke-Roy-IBM - I took a quick look and didn't see your name listed as having filed a CLA; please correct me if I missed it. If you haven't done a CLA yet, please follow the instructions at https://github.com/apache/openwhisk/blob/master/CONTRIBUTING.md.

**Comment by Luke-Roy-IBM** [comment]
@dgrove-oss Thank you for making me aware of the CLA I will get right on it.

**Comment by dgrove-oss** [comment]
@Luke-Roy-IBM please rebase this on latest master to pick up the fixes from #139.  Thanks

**Comment by dgrove-oss** [comment]
Thanks for filing the CLA;  I see Luke Roy listed in http://people.apache.org/unlistedclas.html now.  As soon as we rebase and get a clean run from travis we can merge this.

**Comment by Luke-Roy-IBM** [comment]
@dgrove-oss Thanks for the notice I have rebased my branch.

**Comment by rabbah** [comment]
We're missing in this PR the scala tests for the 5.4 runtime.

**Pull request merged**